literal/decimal_util_test.go

```diff
@@ -0,0 +1,122 @@
+package literal
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_decimalStringToBytes(t *testing.T) {
+	tests := []struct {
+		input        string
+		want         string
```

Comment:
would be clearer if that had been called something like hexwant. took a minute to grok.